learn_script.py

```diff
@@ -172,8 +187,7 @@ def main():
         ansiprint(
             "<bold><light-blue>Running script [{}/{}] - <yellow>'{}'</yellow></light-blue>\n\n".format(
```

Comment:
remove the end="" and the double `\n`:
```suggestion
            "<bold><light-blue>Running script [{}/{}] - <yellow>'{}'</yellow></light-blue>".format(
```